scss/partials/_utility.scss

```diff
@@ -24,15 +24,45 @@
   }
 }
 
-/** 3 - Accessibility
+/** 3 - Accessibility and Visibility
 ------------------------------------------------------------*/
+.hidden {
+  display: none;
+}
+
+.hidden-for-desktop {
+  display: relative;
+  @include breakpoint(desktop) {
+    display: none;
+  }
+}
+
 .show-for-sr {
-  position: absolute !important;
-  width: 1px !important;
-  height: 1px !important;
-  padding: 0 !important;
+  position: absolute;
+  width: 1px;
```

Comment:
Let's make this a rem unit.

Reply:
This, specifically, we don't want in rems, because we don't want it to be able to scale up if the user zooms. It might make for weirdly visible things that should be hidden. Like, you might see the corner of some text peaking in.

Reply:
Cool, understood. 👍🏽